app/views/layouts/GovukLayoutWrapper.scala.html

```diff
@@ -66,6 +65,8 @@
 <!--[if lte IE 8]><link href='@controllers.routes.Assets.versioned("stylesheets/application-ie-8.css")' rel="stylesheet" type="text/css" /><![endif]-->
 <!--[if gt IE 8]><!--><link href='@controllers.routes.Assets.versioned("stylesheets/application.css")' media="all" rel="stylesheet" type="text/css" /><!--<![endif]-->
 
+@hmrcTrackingConsentSnippet()
```

Comment:
This may be a placebo change.  But it seems to reduce the occurrence of VoiceOver on Chrome failing to read out the error messages in the summary.